browser/react/components/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default (props) => (
+  <div style={props.styles.container}>
+    <div>
+      <Link to="signup" style={{ 'textDecoration': 'none' }}>
+        <button key="signup" style={props.styles.signupButton}>Sign Up</button>
```

Comment:
I see you've discovered Radium's annoying thing about needing a key ;)

(Nothing to change here, just commenting lol)